lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -1351,17 +1350,12 @@ void amf_app_handle_initial_context_setup_rsp(
             4);
 
         OAILOG_DEBUG(
-            LOG_AMF_APP,
-            "IP address %02x %02x %02x %02x  and TEID %02x "
-            "%02x %02x %02x \n",
+            LOG_AMF_APP, "IP address %02x %02x %02x %02x  and TEID %02x\n",
```

Comment:
Typo, should not be %02X, you may introduce something like #define TUN_ID_FMT  "%08x" (just an example) to avoid errors and harmonize the logging

Reply:
Comment addressed.